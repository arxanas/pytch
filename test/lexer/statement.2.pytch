foo(
  bar(
    baz))
qux()
grault()
