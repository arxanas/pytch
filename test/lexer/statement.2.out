identifier 'foo'
'('
trailing newline '\n'
leading whitespace '  '
identifier 'bar'
'('
trailing newline '\n'
leading whitespace '    '
identifier 'baz'
')'
')'
trailing newline '\n'
the end of a statement
identifier 'qux'
'('
')'
trailing newline '\n'
the end of a statement
identifier 'grault'
'('
')'
trailing newline '\n'
the end of the file
